src/commands/issuance/create-config.ts

```diff
@@ -0,0 +1,179 @@
+import { readFile } from 'fs/promises'
+import {
+  IssuanceConfigDto,
+  CreateIssuanceConfigInput,
+  CredentialSupportedObject,
+} from '@affinidi-tdk/credential-issuance-client'
+import { WalletDto, CreateWalletInput } from '@affinidi-tdk/wallets-client'
+import { input, select } from '@inquirer/prompts'
+import { ux, Flags } from '@oclif/core'
+import { CLIError } from '@oclif/core/errors'
+import z from 'zod'
+import { BaseCommand } from '../../common/base-command.js'
+import { DidMethods } from '../../common/constants.js'
+import { promptRequiredParameters } from '../../common/prompts.js'
+import { INPUT_LIMIT, validateInputLength } from '../../common/validators.js'
+import { issuanceService } from '../../services/affinidi/cis/service.js'
+import { cweService } from '../../services/affinidi/cwe/service.js'
+
+export class CreateIssuanceConfig extends BaseCommand<typeof CreateIssuanceConfig> {
+  static summary = 'Creates credential issuance configuration in your active project'
+  static examples = [
+    '<%= config.bin %> <%= command.id %> -n <value> -w <value> -f credentialSchemas.json',
+    '<%= config.bin %> <%= command.id %> --name <value> --wallet-id <value> --description <value> --credential-offer-duration <value> --file credentialSchemas.json',
+  ]
+  static flags = {
+    name: Flags.string({
+      char: 'n',
+      summary: 'Name of the credential issuance configuration',
+    }),
+    description: Flags.string({
+      char: 'd',
+      summary: 'Description of the credential issuance configuration',
+    }),
+    'wallet-id': Flags.string({
+      char: 'w',
+      summary: 'ID of the wallet',
+    }),
+    'credential-offer-duration': Flags.integer({
+      summary: 'Credential offer duration in seconds',
+    }),
+    file: Flags.string({
+      char: 'f',
+      summary:
+        'Location of a json file containing the list of allowed schemas for creating a credential offer. One or more schemas can be added to the issuance. The credential type ID must be unique',
+    }),
```

Comment:
How can the developer figure out the structure of this file?
I think we will end up requiring separate CRUD commands for credential configs as we are now also adding metadata display fields to these.

Reply:
That is a good point! We might need to cover it for all places where "file" is an input.